packages/forms-web-app/src/routes/index.js

```diff
@@ -17,6 +17,6 @@ router.use('/cookies', cookieRouter);
 router.use('/appellant-submission', checkDecisionDateDeadline, appellantSubmissionRouter);
 router.use('/eligibility', checkDecisionDateDeadline, eligibilityRouter);
 router.use('/your-planning-appeal', yourPlanningAppealRouter);
-router.use('/before-you-start', beforeYouStartRouter);
+router.use('/before-you-start', fullPlanningRouter);
```

Comment:
Should this not be full-planning?

Reply:
Yes, not sure how anything passed with that on.